### DISCUSSION TITLE
I had a problem related to plot_3d while learning the example, how should I solve it, I put it in the following picture

### DISCUSSION THREAD

**Discussion by yu5213** [gempy-project/gempy#789] [General]
![image](https://user-images.githubusercontent.com/101611838/231164150-4d5b3685-33c2-4929-be1c-756fbbb1b6eb.png)

**Comment by AlexanderJuestel** [comment]
See https://github.com/cgre-aachen/gempy/issues/754 
A workaround would be to downgrade PyVista
